src/pages/vedtak-side/sykepengedager/sykepengedager.tsx

```diff
@@ -0,0 +1,67 @@
+import dayjs from 'dayjs'
+import parser from 'html-react-parser'
+import { Normaltekst, Systemtittel } from 'nav-frontend-typografi'
+import React, { useState } from 'react'
+
+import Utvidbar from '../../../components/utvidbar/utvidbar'
+import { useAppStore } from '../../../data/stores/app-store'
+import { erHelg } from '../../../utils/dato-utils'
+import { tekst } from '../../../utils/tekster'
+import LedningImg from './ikon-sykefravaersoversikt.svg'
+
+const Sykepengedager = () => {
+    const { valgtVedtak } = useAppStore()
+    const [ apen ] = useState<boolean>(false)
+
+    const kalkulerSluttdato = () => {
+        let slutt = dayjs(valgtVedtak!.vedtak.tom)
+        let x = 0
+        while (x < valgtVedtak!.vedtak.gjenståendeSykedager) {
+            slutt = slutt.add(-1, 'day')
```

Comment:
Denne og den under må telle oppover, siden sluttdato er en gang i fremtiden.

Reply:
Det burde kanskje også legges inn en egen test som tester grenseverdier (like før og etter helg) for sluttdato

Reply:
+1. tenker det er en fin sak for enhetstest, ikke cypress